### PR TITLE
SEO update advanced-hunting-overview

### PR DESCRIPTION
**Pull request by tuco-ulf**
replaces PR#2249
reverted to original metadata title
added high value search keyword to description
added "cyber threat" keyword to body
@DaniHalfin

### CODE REVIEW

**Comment by DaniHalfin** [comment]
@lomayor - ok to merge?

File: microsoft-365/security/mtp/advanced-hunting-overview.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Overview - Advanced hunting
-description: Learn about advanced hunting queries in Microsoft 365 and how to use them to proactively find threats and weaknesses in your network
+description: Learn about advanced hunting queries in Microsoft Threat Protection (MTP) and how to use them to proactively find threats and weaknesses in your network
```
Comment (lomayor) [inline microsoft-365/security/mtp/advanced-hunting-overview.md]: MTP is not an approved short form

File: microsoft-365/security/mtp/advanced-hunting-overview.md [hunk 2]
```diff
@@ -27,9 +27,9 @@ ms.custom: seo-marvel-apr2020
 
 [!INCLUDE [Prerelease information](../includes/prerelease.md)]
 
-Advanced hunting is a query-based threat-hunting tool that lets you explore up to 30 days of raw data. You can proactively inspect events in your network to locate interesting indicators and entities. The flexible access to data facilitates unconstrained hunting for both known and potential threats.
+Advanced hunting is a query-based cyber threat hunting tool that lets you explore up to 30 days of raw data. You can proactively inspect events in your network to locate interesting indicators and entities. The flexible access to data facilitates unconstrained hunting for both known and potential threats.
 
-You can use the same threat-hunting queries to build custom detection rules. These rules run automatically to check for and respond to various events and system states, including suspected breach activity and misconfigured machines.
+You can use the same cyber threat hunting queries to build custom detection rules. These rules run automatically to check for and respond to various events and system states, including suspected breach activity and misconfigured machines.
```
Comment (lomayor) [inline microsoft-365/security/mtp/advanced-hunting-overview.md]: Revert. "cyber threat hunting" does not make sense as a modifier.

**Comment by tuco-ulf** [comment]
@DaniHalfin reverted edits

**Pull request closed without merging**